### DISCUSSION TITLE
How work pro plan ? I noticed that Compute Billing is automatically set to $10 every time

### DISCUSSION THREAD

**Discussion by tcouet** [supabase/supabase#29217] [Questions] [answered]
![CleanShot 2024-09-11 at 09 09 37](https://github.com/user-attachments/assets/092a9742-629b-4664-b3c9-62547376ba03)

Hello,

I recently upgraded to the Pro plan to avoid the limitation of having only two projects. However, after switching to the Pro plan and adding another project, I noticed that Compute Billing is automatically set to $10, even though I'm already paying $25 per month for the Pro plan. Could you please explain why this happens, how I can avoid these additional charges, and how this works in general?

Thank you for your help.

Best regards,

**Comment by GaryAustin1** [comment, marked as answer]
The smallest Pro instances use about $10 of compute a month.  You get a $10 credit as part of your $25 to cover the first pro instance.  If you add more instances to that organization then you have to pay for their compute.

**Reply by GaryAustin1** [reply]
Note you can still have 2 free plans in free organizations, but not in a Pro organization.

**Reply by tcouet** [reply]
Then I need to pay 10$ minimum for each New project on Pro ?  🧐

**Reply by GaryAustin1** [reply]
Yes, except for your 2 free ones in non pro orgs.

**Reply by tcouet** [reply]
Ok, thanks for answer. then its really expansive for small project
